examples/firebase_example/tools/bazel

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+# Order of Bazel selection:
+# 1. BAZEL_REAL: Use Bazel specified by Bazelisk
+# 2. BIT_BAZEL_BINARY: Use Bazel specified by rules_bazel_integration_test.
+# 3: bazel: Try to find Bazel on the PATH
```

Comment:
Should this just fail? Might be clearer to enforce bazelisk usage if we require a specific Bazel version

Reply:
> Might be clearer to enforce bazelisk usage if we require a specific Bazel version

Are we requiring a specific Bazel version? This PR is merely ensuring that `rules_xcodeproj` uses the Bazel that was used by any parent processes.